src/components/MoviePoster.js

```diff
@@ -0,0 +1,13 @@
+import React from "react";
+import "../css/MoviePoster.css";
+
+const MoviePoster = ({ title, rating, poster }) => {
+  return (
+    <section className="poster-frame">
+      <img src={poster} alt="movie" />
+      <p>Rating: {parseFloat(rating).toFixed(1)} / 10</p>
```

Comment:
We will clean the data before passing it through once we are further along in the project.